docs/conceptual-guides/package-management.md

```diff
@@ -59,6 +59,10 @@ You or your development tools, such as GNU Make, must perform that step.
 
 ## Manage frontend Node.js packages
 
-```{todo}
-Why do we use pnpm?
-```
```

Comment:
Revert, create an issue if one does not already exist either in Documentation or Volto repo, and add its link here.